pytest_selenium/pytest_selenium.py

```diff
@@ -265,6 +262,27 @@ def _gather_logs(item, report, driver, summary, extra):
                 format_log(log), '%s Log' % name.title()))
 
 
+def _gather_driver_logs(item, summary, extra):
+    pytest_html = item.config.pluginmanager.getplugin('html')
+    try:
+        item.config._driver_log
+    except Exception:
+        summary.append('No Driver Log found')
```

Comment:
Rather than try except, let's just use `if item.config._driver_log and os.path.exists(item.config._driver_log)`

Reply:
With this, if ```item.config._driver_log``` doesn't exist, it returns ```None``` and then ```os.path.exists``` tries to find that and throws a NoneType error.

Reply:
If `item.config._driver_log` is `None` then it is not truthy, so the following condition should not be checked.

Reply:
Hmm, it seemed like that wasn't happening but I'll check again.